src/stories/3-fields/date-field/date-field.stories.tsx

```diff
@@ -77,6 +77,18 @@ const meta: Meta = {
 				type: "boolean",
 			},
 		},
+		withinDays: {
+			description: "Validate if value is within x days from now or specific date",
```

Comment:
explain that if a specific date is provided, the format will follow the `dateFormat` option

Reply:
sorry, my bad, pls remove the `withinDays` from the stories because it is a validation rule but these descriptions are for props.